dbms/src/DataStreams/RuntimeFilter.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "RuntimeFilter.h"
+
+#include <Interpreters/Set.h>
+#include <Storages/DeltaMerge/FilterParser/FilterParser.h>
+#include <common/logger_useful.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+extern const int SET_SIZE_LIMIT_EXCEEDED;
+}
+
+std::string RuntimeFilter::getSourceColumnName()
+{
+    return source_column_name;
+}
```

Comment:
can be marked as const?

```suggestion
std::string RuntimeFilter::getSourceColumnName() const
{
    return source_column_name;
}
```